python/ray/data/dataset.py

```diff
@@ -3390,7 +3390,7 @@ def write_parquet(
                 opening the file to write to.
             filename_provider: A :class:`~ray.data.datasource.FilenameProvider`
                 implementation. Use this parameter to customize what your filenames
-                look like.
+                look like. The following convention is used: ```{filename}-{i}.parquet```
```

Comment:
This is not sufficient

 - Explain its a template 
 - Explain what i refers to in here